### PR TITLE
Closes softlayer/sl-ember-components#512

### PR DESCRIPTION
**Pull request by theoshu**
added return null to sl-grid computed properties

### CODE REVIEW

File: addon/components/sl-grid.js [hunk 1]
```diff
@@ -665,7 +669,14 @@ export default Ember.Component.extend({
         'content.length',
         'totalCount',
         function() {
```
Comment (notmessenger) [inline addon/components/sl-grid.js]: `@returns` value needs updated

**Comment by notmessenger** [comment]
Should `totalPages()` be modified?

**Comment by notmessenger** [comment]
No tests needed to be modified?

File: addon/components/sl-grid.js (hunk 1 above)
Comment (notmessenger) [inline addon/components/sl-grid.js]: Does this computed property need to return `null`?  Seems `Boolean` is the only valid return value?

**Comment by theoshu** [comment]
@notmessenger re: your question about `totalPages()`, are you asking why did I change it?

**Comment by notmessenger** [comment]
@theoshu Regarding your question about my `totalPages()` comment: No, my comment was regarding that there we no changes made to `totalPages()` and perhaps there needs to be?  Upon further review however, specifically line https://github.com/theoshu/sl-ember-components/blob/issue512/addon/components/sl-grid.js#L499, if the value of `totalPages()` is ever anything undesired, this line will ensure correct behavior of the component.

File: addon/components/sl-grid.js (hunk 1 above)
Comment (theoshu) [inline addon/components/sl-grid.js]: fixed

File: addon/components/sl-grid.js [hunk 2]
```diff
@@ -546,6 +548,8 @@ export default Ember.Component.extend({
                     this.get( 'totalCount' ) / this.get( 'pageSize' )
                 );
             }
+
+            return null;
```
Comment (notmessenger) [inline addon/components/sl-grid.js]: Change DocBlock `@returns {Number|undefined}` to `@returns {?Number}`

**Comment by notmessenger** [comment]
Why is https://github.com/theoshu/sl-ember-components/blob/issue512/tests/unit/components/sl-grid-test.js#L323 and https://github.com/theoshu/sl-ember-components/blob/issue512/tests/unit/components/sl-grid-test.js#L335 being set to the same value twice?  Doesn't seem like https://github.com/theoshu/sl-ember-components/blob/issue512/tests/unit/components/sl-grid-test.js#L326 and https://github.com/theoshu/sl-ember-components/blob/issue512/tests/unit/components/sl-grid-test.js#L338 are needed twice either.

File: addon/components/sl-grid.js (hunk 2 above)
Comment (theoshu) [inline addon/components/sl-grid.js]: Done

**Comment by notmessenger** [comment]
`showPagination()`, 'sortedColumn()`, and `totalPages()` should be in the "Methods" section.

File: addon/components/sl-grid.js (hunk 1 above)
Comment (notmessenger) [inline addon/components/sl-grid.js]: RESOLVED

**Pull request merged**